sdk/core/src/CommonNetworkMember.ts

```diff
@@ -1522,8 +1522,12 @@ export class CommonNetworkMember {
     const credentialTypes = []
 
     for (const credential of credentialRequirements) {
-      const type = credential.type[1] // seems its always next structure type: ['Credential', 'ProofOfEmailCredential|....']
-      credentialTypes.push(type)
+      const isW3cCredential = credential.type
```

Comment:
@rbrazhnyk lets have isW3cCredential as method or helper and use it in each proper places, so we will have one place which we will need to change when will need change definition on how to detect is that w3c credential or not